.github/workflows/backup-clever.yml

```diff
@@ -0,0 +1,14 @@
+name: "Backup to clever"
```

Comment:
_nit_ : pour ma part, pour les messages je préfère _"in french in the text"_.

Reply:
ok pas de souci